polyglot/piranha/test-resources/go/feature_flag/builtin_rules/boolean_expression_simplify/configurations/rules.toml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "true_flag"
```

Comment:
update rule name. Pass flag behavior as a piranha argument.

Reply:
@ketkarameya `boolean_expression_simplify` has a single file exercising replacements for `true`, `false`, and `nil`.
Should I split it into 3 directories/scenarios to keep the calls to `exp.BoolValue`?

Reply:
Ideally we should templatize `piranha_arguments.toml` , but I guess that infra is not available at this moment. Let this be. Can we do this as a follow up?